File: Python/Einsteiger/uebung2a.py
"""
1.    Schreiben Sie ein Programm, das den Anwender dazu auffordert, zwei Zahlen einzugeben.
      Speichern Sie diese in zwei unterschiedlichen Variablen.
      Das Programm soll danach die beiden Werte addieren und ausgeben.

Bonacina, Michael. Python 3: Programmieren für Einsteiger: Der leichte Weg zum Python-Experten (Einfach Programmieren lernen 2) (German Edition) (S.54). Kindle-Version.
"""


def main():
    x = eval(input("Geben Sie eine erste Zahl ein: "))
    y = eval(input("Geben Sie eine zweite Zahl ein: "))

    z = x + y
    print("Summe: {}".format(z))

if(__name__ == "__main__"):
    main()

File: Java/Einsteiger/uebung2a.java
/*
1.  Gestalten Sie ein Programm, das die Zahlen 3 und 3,5 jeweils in einer Variablen speichert und die Werte anschließend addiert.
    Das Ergebnis soll in einer weiteren Variablen als Fließkommazahl gespeichert und anschließend auf dem Bildschirm ausgegeben werden.
    Wiederholen Sie diesen Vorgang, speichern Sie dabei dieses Mal jedoch nur den ganzzahligen Teil in der Variablen ab.

Bonacina, Michael. Java Programmieren: für Einsteiger: Der leichte Weg zum Java-Experten (2. Auflage: komplett neu verfasst) (Einfach Programmieren lernen 1) (German Edition) (S.58). Kindle-Version.

*/

public class uebung2a
{
    public static void main(final String[] args)
    {
        int x = 3;
        double y = 3.5;

        double z1 = x + y;
        System.out.printf("Ergebnis: %.1f\n", z1);

        int z2 = (int) (x + y);
        System.out.printf("Ergebnis: %d", z2);
    }
}